dbms/src/Flash/tests/gtest_projection_executor.cpp

```diff
@@ -44,17 +44,17 @@ class ExecutorProjectionTestRunner : public DB::tests::ExecutorTest
     }
 
     template <typename T>
-    std::shared_ptr<tipb::DAGRequest> buildDAGRequest(T param, const String & sort_col)
+    std::shared_ptr<tipb::DAGRequest> buildDAGRequest(T param)
     {
         /// topN is introduced, so that we can get stable results in concurrency environment.
-        return context.scan(db_name, table_name).project(param).topN(sort_col, false, 100).build(context);
```

Comment:
why remove `topN(sort_col, false, 100)`?

Reply:
It seems `topN` only aims to get stable results instead of testing its correctness. Since now using `ASSERT_COLUMNS_EQ_UR` we can correctly test result even in unstable situation. It might be better to remove `topN` and make the ut more compact? I think I might forget to also remove the comment...

Reply:
And `TopN` of https://github.com/pingcap/tiflash/blob/cbe6ab5593926ba82d15f8b2ac398fd98dff56bc/dbms/src/Flash/tests/gtest_executor.cpp#L94, https://github.com/pingcap/tiflash/blob/cbe6ab5593926ba82d15f8b2ac398fd98dff56bc/dbms/src/Flash/tests/gtest_executor.cpp#L126, https://github.com/pingcap/tiflash/blob/cbe6ab5593926ba82d15f8b2ac398fd98dff56bc/dbms/src/Flash/tests/gtest_executor.cpp#L144, https://github.com/pingcap/tiflash/blob/cbe6ab5593926ba82d15f8b2ac398fd98dff56bc/dbms/src/Flash/tests/gtest_executor.cpp#L174, https://github.com/pingcap/tiflash/blob/cbe6ab5593926ba82d15f8b2ac398fd98dff56bc/dbms/src/Flash/tests/gtest_executor.cpp#L211 can be removed. 
/cc @ywqzzy

Reply:
This might not be a problem.. I am working on moving these join tests to a new separate test file.